### DISCUSSION TITLE
Be able to bulk remove issues from Project boards (beta)

### DISCUSSION THREAD

**Discussion by drbilo** [community/community#10521] [Projects and Issues]
~~Unless I'm mistaken you can only remove an issue from a project by navigating to the issue itself and removing~~. I'd like to be able to remove multiple tickets in the project view at a time rather than one by one. Also the language `Delete issue from project` is a little scary, consider changing to `Remove issue from project`

**Comment by drbilo** [comment]
Can close as discovered https://github.com/github/feedback/discussions/5465

**Comment by sakib789** [comment]
Hi @drbilo 

I was looking at the GitHub thread you have mentioned but count figure out the solution. Can you guide me

**Comment by markcaron** [comment]
Any update on how to do this? With the frustrating Project limit of 2,000 issues, I'm finding myself needing to bulk remove issues (rather than go one by one) so I can make room for my team to keep working.

**Comment by ianhinder** [comment]
This seems to be possible now.
1. Go to a table view which contains the issues you want to remove.
2. Select the issues you want to remove by clicking/ctrl-clicking/shift-clicking etc. The entire rows should be blue (not just individual cells).
3. Click the "Row actions" down-arrow button that appears at the left of any of the rows while you are hovering over a row.
4. From the menu that appears, choose "Delete NN selected items from project". I guess you can also press Del on the keyboard.

![image](https://github.com/user-attachments/assets/1c9ad7e2-726f-41f6-9139-695943768216)

**Comment by dzimine-lc** [comment]
Doesn't seem to work for me, when I do it it only offers me to delete one selected issue
![image](https://github.com/user-attachments/assets/d8b3b989-c45e-49fe-88ea-bf781c61d2f9)

**Reply by sheridn** [reply]
Same here

**Comment by ianhinder** [comment]
I also ran into this, but when I tried it a second time, it then worked. It might be that the page needs to "catch up" to realise that multiple things are selected. GitHub Projects is not very good at refreshing state, and is really slow and clunky, and I wouldn't be surprised if this was just a poor implementation. The feature is definitely there, as I have used it.

**Reply by aikenka** [reply]
Finally got this working for me!

When doing the multi-select - hover more over the line number than the item title; the cursor (for me anyway) is a hand rather than an arrow.

key point from @ianhinder - 

> The entire rows should be blue (not just individual cells).

![20250916_172257](https://github.com/user-attachments/assets/1c755b73-acfc-4cdd-b3d0-a2dd9e85c2a8)

**Comment by sheridn** [comment]
This works for me as well, actually. I now understand what @ianhinder meant in his comment:
> The entire rows should be blue (not just individual cells).

You have to click on the row number when multi-selecting, as @aikenka said. You can tell you've selected the entire row if the whole row is highlighted, not just the title cell of each row.

Should look like this (full row highlighted):
<img width="736" height="239" alt="Screenshot 2025-09-17 at 10 03 05 AM" src="https://github.com/user-attachments/assets/901656ec-9707-4e1a-81fe-510f9b229795" />

_Not_ this (only title cell highlighted):
<img width="733" height="240" alt="Screenshot 2025-09-17 at 10 06 01 AM" src="https://github.com/user-attachments/assets/f7cdbd9f-4482-4339-b853-a7c07766e4da" />

Thanks to those who persevered in their explanations on this thread!
